tensorflow/python/ops/nn_ops.py

```diff
@@ -682,16 +682,14 @@ def convolution(
   of N `strides` (defaulting [1]*N), this computes for each N-D spatial output
   position (x[0], ..., x[N-1]):
 
-  ```
     output[b, x[0], ..., x[N-1], k] =
-        sum_{z[0], ..., z[N-1], q}
+        \\(sum_{z[0], ..., z[N-1], q}\\)
```

Comment:
These equations look longer but your change only converts 1 line. Should entire equations be surrounded by \\( \\)? How would it look if only 1 line is converted?